objects/src/accounts/storage/mod.rs

```diff
@@ -241,10 +241,15 @@ impl AccountStorage {
     }
 
     /// Returns the storage maps for this storage.
-    pub fn maps(&self) -> &[StorageMap] {
+    pub fn maps(&self) -> &BTreeMap<u8, StorageMap> {
         &self.maps
     }
 
+    /// Returns the mutable storage map with a given index.
+    pub fn get_storage_map_by_index(&mut self, index: u8) -> Option<&mut StorageMap> {
+        self.maps.get_mut(&index)
+    }
```

Comment:
We shouldn't need this method. We should be able to route all updates to storage maps via `set_map_item()` or `apply_delta()`.

Reply:
I removed that function